### DISCUSSION TITLE
Do not auto-start RustDesk on Windows (disable system tray icon)

### DISCUSSION THREAD

**Discussion by rkrisztian** [rustdesk/rustdesk#10400] [General]
This feature is very important to me, especially because TeamViewer does not work the same way. On Windows, I don't want the RustDesk service to start automatically with the operating system. Instead, **I want the service to run only while I have the RustDesk client opened**. This behavior aligns with how the Linux client works and matches my expectations for the Windows client as well.

To make RustDesk as user-friendly as possible for my family members, it should be simple to start and stop the application along with its service. Ideally, this should be as easy as clicking a desktop shortcut to launch it and closing the application by clicking the "X" button in the window's title bar.

**Comment by dojoca** [comment]
Just adding my request to this one.

**Comment by PeterLairo** [comment]
Please see bug #11060.

### Add setting UI to disable autostart of RustDesk in Startup (Taskbar) (Windows)

**Reply by BrancuAlexandru** [reply]
the issue has been deleted

**Reply by zbyna** [reply]
Any info on this question will be appreciated. Was that that deleted bug #11060 fixed?

**Comment by open-trade** [comment]
Turn on this option. Go to `Settings` -> `Security`.

<img width="864" height="110" alt="1396ce09-dbf8-48d6-a64d-9cc511804bbb" src="https://github.com/user-attachments/assets/5a593c4d-5e16-4577-9fcd-d7f7e1f01f24" />

**Reply by open-trade** [reply]
Another two ways.

- Do not install it
- If you installed, please stop the rustdesk service with command `sc stop RustDesk`
